### PR TITLE
centralize VS properties for target image name and tag

### PR DESCRIPTION
**Pull request by baronfel**
Fill in target image name as given by @vijayrkn, and centralize all the VS-specific defaults in one location for easier review.

### CODE REVIEW

File: packaging/build/Microsoft.NET.Build.Containers.targets [hunk 1]
```diff
@@ -29,6 +29,13 @@
             <_ContainerBaseImageTag>$(_TargetFrameworkVersionWithoutV)</_ContainerBaseImageTag>
         </PropertyGroup>
 
+        <PropertyGroup Label="VS defaults">
+            <!-- OutputRegistry is used by existing VS targets for Docker builds - this lets us fill that void -->
+            <ContainerRegistry Condition="'$(OutputRegistry)' != ''">$(OutputRegistry)</ContainerRegistry>
```
Comment (vijayrkn) [inline packaging/build/Microsoft.NET.Build.Containers.targets:34]: VS uses RegistryUrl

Comment (baronfel) [reply packaging/build/Microsoft.NET.Build.Containers.targets]: thanks, updated!

**Review by vijayrkn — APPROVED** [review]

**Review by rainersigwald — APPROVED** [review]
Should we switch to these property names entirely?

**Comment by baronfel** [comment]
I think I like having our own, consistent names for things. That leaves us in control.

**Pull request merged**